components/PixTypography.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <component :is="tag"><slot /></component>
+  <component :is="tag" :class="`pix-${scale}`"><slot /></component>
```

Comment:
```suggestion
  <component :is="tag" :class="`pix-typography pix-typography--scale-${scale}`"><slot /></component>
```

Pour mieux respecter le BEM ?